azurelinuxagent/common/cgroupapi.py

```diff
@@ -67,9 +67,9 @@ def track_cgroups(extension_cgroups):
                         "Error: {1}".format(cgroup.path, ustr(exception)))
 
     @staticmethod
-    def _get_extension_cgroup_name(extension_name):
+    def get_extension_cgroup_name(extension_name):
```

Comment:
the slice names are particular to systemd, so let's make this change in SystemdCgroupsApi instead of CGroupsApi

Reply:
Updated.